db/seeds.rb

```diff
@@ -54,6 +54,15 @@
   info_url: 'http://www.blackgirlscode.com',
   donation_url: 'http://www.blackgirlscode.com'
 }, {
+  name: 'Hillary Clinton Campaign',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.